gorm/transaction.go

```diff
@@ -17,15 +18,18 @@ import (
 // ctxKey is an unexported type for keys defined in this package.
 // This prevents collisions with keys defined in other packages.
 type ctxKey int
+type readOnlyDBKey int
```

Comment:
We have comments for existing fields, we should keep this pattern when adding new ones.